app/src/main/java/com/jshvarts/healthreads/ui/bookdetail/BookDetailFragment.kt

```diff
@@ -74,20 +69,18 @@ class BookDetailFragment : Fragment() {
   }
 
   private fun renderLoadingState() {
-    snackbar?.dismiss()
     binding.pullToRefresh.isRefreshing = true
   }
 
-  private fun renderErrorState() {
+  private fun renderErrorState(errorType: ErrorType) {
     binding.pullToRefresh.isRefreshing = false
 
-    snackbar = Snackbar.make(
-      binding.root,
-      R.string.error_message,
-      Snackbar.LENGTH_INDEFINITE
-    ).setAction(R.string.refresh_button_text, refreshOnErrorListener).also {
```

Comment:
`Snackbar` with a refresh button seemed redundant with a pull to refresh ux. removing it, removed quite a bit of code